app/views/groups/show.html.erb

```diff
@@ -184,6 +184,7 @@
               this.innerHTML = formatDate(new Date(this.innerHTML), "dddd hh:mmTT zz, dd MMM yyyy")
           })
       })
+      setTimeout(()=>document.querySelector(".alert").remove(), 3000);
```

Comment:
I think these alerts are used everywhere, login, project dashboard etc. Could you confirm the same? If yes, maybe move this to `/app/views/layouts/application.html.erb`

Reply:
Please use `hide` instead of `remove`, we don't want to reload the element in DOM again if needed.
And also what @satu0king said.

Reply:
@satu0king, yeah I've just noticed that, working on it.

Reply:
@armudgal nice remark, will change it now.